wasm/demo/webpack/index.js

```diff
@@ -0,0 +1,61 @@
+// Note that a dynamic `import` statement here is required due to
+// webpack/webpack#6615, but in theory `import { greet } from './pkg';`
+// will work here one day as well!
+import * as cesride from 'cesride-wasm';
+
+document.write(`
+<style>
+  body {
+    background-color: black;
+    color: #cccccc;
+  }
+</style>
+`)
+
+let dts = "2020-08-22T17:50:09.988921+00:00";
+let dater = cesride.Dater.new_with_dts(dts);
+
+const icp = {
+  "v": "KERI10JSON00015a_",
+  "t": "icp",
+  "d": "EBAjyPZ8Ed4XXl5cVZhqAy7SuaGivQp0WqQKVXvg7oqd",
+  "i": "BEy_EvE8OUMqj0AgCJ3wOCOrIVHVtwubYAysPyaAv9VI",
+  "s": "0",
+  "kt": "1",
+  "k": [
+    "BEy_EvE8OUMqj0AgCJ3wOCOrIVHVtwubYAysPyaAv9VI"
+  ],
+  "nt": "0",
+  "n": [],
+  "bt": "2",
+  "b": [
+    "BC9Df6ssUZQFQZJYVUyfudw4WTQsugGcvVD_Z4ChFGE4",
+    "BEejlxZytU7gjUwtgkmNKmBWiFPKSsXjk_uxzoun8dtK"
+  ],
+  "c": [],
+  "a": []
+}
+
+const raw = new TextEncoder().encode(JSON.stringify(icp))
+const serder = cesride.Serder.new_with_raw(raw)
```

Comment:
I think I was mistaken making this `new_with_raw()` wrapping in the spike. You can probably use new with optional arguments. The `*_with_*()` functions in `cesride` are convenient for Rust where you need to define all optional arguments.